types/images/index.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module '*.png' {
+  const value: any
+  export = value;
+}
+
+declare module '*.svg' {
+  const value: any
```

Comment:
I'm pretty sure there is a nice type for this that maps to the svg structure expected inside

Reply:
Did you find anything with this?
Is this something @nawatts knows about?

Reply:
I think the default export should be `string` instead of `any`. The default export is the URL for the image generated by webpack. SVG files also have a `ReactComponent` named export.

Reply:
String worked for me!